PyRDF/RDataFrame.py

```diff
@@ -16,12 +16,10 @@ class RDataFrame(Node):
 
     Supported constructor arguments
     -------------------------------
-    - number of entries (int)
-        * For eg., RDataFrame(64)
-    - treename (str), fileslist (str or list of str)
-        * For eg., RDataFrame("myTree", "file.root") or RDataFrame("myTree", ["file1.root", "file2.root"])
-    - treename (str), fileslist (str or list of str or vector of str), default branches (list of str or vector of str)
-        * For eg., RDataFrame("myTree", ["file1.root", "file2.root"], ["branch1", "branch2"])
+    PyRDF's RDataFrame constructor accepts the same arguments as the ROOT's RDataFrame
+    constructor (see https://root.cern/doc/master/classROOT_1_1RDataFrame.html). In addition,
+    PyRDF allows you to use Python lists in place of C++ vectors as arguments of the constructor,
+    for e.g., RDataFrame("myTree", ["file1.root", "file2.root"].
```

Comment:
for e.g. is redundant, either for example or e.g.
Also, can we put the RDataFrame code in between backticks?

Reply:
Sure @etejedor . Will change it right away 👍